src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -609,6 +609,7 @@ private static Properties loadProperties(File file)
      * @return a fresh new {@code AuditListener}
      * @exception IOException when provided output location is not found
      */
+    @SuppressWarnings("resource")
```

Comment:
Please explain suppress reason in javadoc.
Why IDEA does not have there suppression ? Is it a hug in Eclipse analyser?

Reply:
ok, call it new feature request, enhancement, any type of ticket against Eclipse to be better.
If IDEA found way to make less false positives, why Eclipse should be behind ?

If Eclipse is less configurable/smart ... why we should activate it ? Does it make sense to activate only IDEA ?